src/web/mjs/connectors/MangaProTM.mjs

```diff
@@ -4,10 +4,10 @@ export default class MangaProZ extends WordPressMangastream {
 
     constructor() {
         super();
-        super.id = 'mangaproz';
-        super.label = 'Manga Pro Z';
+        super.id = 'mangaprotm';
```

Comment:
I suggest to keep the old `ID`, otherwise users who have bookmarked this website must manually remove and re-add the bookmarks from this website

Reply:
I just want to change the website link to the new one 
mangaprotm.com

Reply:
I hope you will change it